input_files/strtx_III_dsp/dsp_hb_settings.txt

```diff
@@ -17,20 +17,22 @@ area_cost_exp=1.0
 #ASIC flow settings
 #path to absolute paths needed for process parameters (relative to coffe home repo)
 process_params_file=input_files/strtx_III_dsp/process_specific_params.txt
-
+#run_synth=False

```

Comment:
What does it mean to have all 3 of these false?  Good to comment what these control (which tools are run, so you can run one stage at a time?)

Reply:
Sorry I think there were a few leftover parameters that weren't needed, they'll be removed